### PR TITLE
creating basic express app, using express generator closes #9

### LINKED ISSUE #9: Create basic express app
- server.js
- app.js
- router.js
- handler.js (if needed)

### PR DESCRIPTION
**Pull request by ebraheemabbas**
adding 
server.js / bin/www
app.js
router.js
views 
using express generator closes #9

### CODE REVIEW

**Review by omriza5 — DISMISSED** [review]

**Review by HosenAgbaria — DISMISSED** [review]

**Comment by jamiecoe** [comment]
@ebraheemabbas - In the future, try to give your branch a more relevant name. Eg: `create-basic-express-app`.

This makes it easier for other team members (and yourself in the future) to understand what is on a branch.

File: the-experts-BE/app.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+var logger = require('morgan');
+
+var indexRouter = require('./routes/index');
+var usersRouter = require('./routes/users');
+
+var app = express();
+
+// view engine setup
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'jade');
```
Comment (Karyum) [inline the-experts-BE/app.js]: lets remove this view engine since we are not going to use at all

**Review by Karyum — CHANGES_REQUESTED** [review]
yeah the only change is to remove the jade view engine, with all of the files that it created `error.jade` `index.jade` `layout.jade`

also add a desription to the PR and `close #issueNumberHere`

**Review by Karyum — APPROVED** [review]

**Comment by Karyum** [comment]
oh you need @omriza5 or @later009 to approve to actually merge

**Review by HosenAgbaria — APPROVED** [review]

**Review by omriza5 — APPROVED** [review]

**Pull request merged**